model/license.go

```diff
@@ -224,6 +228,14 @@ func (f *Features) SetDefaults() {
 	if f.Cloud == nil {
 		f.Cloud = NewBool(false)
 	}
+
+	if f.SharedChannels == nil {
+		f.SharedChannels = NewBool(false)
```

Comment:
@nronas if you want this to be enabled for E20 license you should set this to `f.FutureFeatures`. Otherwise old E20 licenses will not get this new feature unless we re-issue them a new license